assets/kube-burner-report-ocp-wrapper/queries.libsonnet

```diff
@@ -552,181 +552,7 @@ local elasticsearch = g.query.elasticsearch;
       + elasticsearch.withQuery('uuid.keyword: $uuid AND metricName.keyword: alert')
       + elasticsearch.withTimeField('timestamp'),
   },
-  ovnKubeMasterPodStats: {
```

Comment:
Just curious, why are we removing these queries?

Reply:
This pod contains ovnkube-cluster-manager container with very minimal functionality and we can ignore metrics for this pod and the container.